kusto/queryopts.go

```diff
@@ -135,14 +135,9 @@ func ResultsProgressiveDisable() QueryOption {
 	}
 }
 
-// queryServerTimeout is the amount of time the server will allow a query to take.
-// NOTE: I have made the serverTimeout private. For the moment, I'm going to use the context.Context timer
-// to set timeouts via this private method.
-func queryServerTimeout(d time.Duration) QueryOption {
+// ServerTimeout overrides the default request timeout.
+func ServerTimeout(d time.Duration) QueryOption {
 	return func(q *queryOptions) error {
-		if d > 1*time.Hour {
```

Comment:
Why?
Is the service contract different?

Reply:
We just don't have this limit in any other SDK, so I removed it for parity

Reply:
In other SDKs though we take the timeout from the additional properties